aws-predictions/src/androidTest/java/com/amplifyframework/predictions/aws/AWSPredictionsIdentifyTextTest.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.aws;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import androidx.core.util.Pair;
+
+import com.amplifyframework.predictions.PredictionsCategory;
+import com.amplifyframework.predictions.PredictionsException;
+import com.amplifyframework.predictions.aws.test.R;
+import com.amplifyframework.predictions.models.BoundedKeyValue;
+import com.amplifyframework.predictions.models.Cell;
+import com.amplifyframework.predictions.models.IdentifyActionType;
+import com.amplifyframework.predictions.models.Table;
+import com.amplifyframework.predictions.models.TextFormatType;
+import com.amplifyframework.predictions.result.IdentifyDocumentTextResult;
+import com.amplifyframework.predictions.result.IdentifyTextResult;
+import com.amplifyframework.testutils.Assets;
+import com.amplifyframework.testutils.FeatureAssert;
+import com.amplifyframework.testutils.sync.SynchronousMobileClient;
+import com.amplifyframework.testutils.sync.SynchronousPredictions;
+import com.amplifyframework.util.CollectionUtils;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static androidx.test.core.app.ApplicationProvider.getApplicationContext;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * Tests that Predictions identify delivers a non-null
+ * text detection result for valid input.
```

Comment:
Hmm, I would consider `IdentifyActionType#DETECT_TEXT` an *invalid* input, so it would still be a true statement. It just isn't a very descriptive of a javadoc haha